src/examples/mfem_mortar_lm_patch.cpp

```diff
@@ -348,6 +357,9 @@ int main( int argc, char** argv )
   // HypreParMatrices holding the Jacobian from contact and stored in an MFEM
   // block operator are returned from this API call.
   auto A_blk = tribol::getMfemBlockJacobian(coupling_scheme_id);
+  // Surface mesh contains mortar and nonmortar surfaces.  Remove pressure DOFs
+  // on mortar surfaces.
+  clearInactivePressureDofs(*A_blk, coupling_scheme_id, nonmortar_attrs);
```

Comment:
Does this clear just the mortar dofs? Or does this also clear inactive dofs that are non-mortar, but not part of the active set? Maybe clarify.

Reply:
Sounds good.  It puts a 1 on the diagonal of mortar dofs in the matrix, but doesn't touch non-mortar inactive dofs.  As implemented, the host code is responsible for updating the active set.  Will update the comment on L359; the `clearInactivePressureDofs()` will be buried in `tribol::getMfemBlockJacobian()` after refactoring.